Laravel/readme.md

```diff
@@ -37,3 +37,15 @@ If that runs out of memory again, you can try using the -`-size=standard-2x` Her
 ```
 heroku run:detached php -d memory_limit=1024M artisan northstar:another-backfill-command --app dosomething-northstar-qa --size=standard-2x
 ```
+
+### Standard Input
+
+We often provide source files for imports as standard input, to "pipe" the file into the command from your local machine. This way, we don't have to worry about hosting it somewhere on the internet. 
+
+Example:
+
+```
+cat ../college-board.csv | heroku run php -d memory_limit=512M artisan rogue:groups-import --name="The College Board" --filterByLocation --app dosomething-rogue-dev --no-tty
+```
+
+The `--no-tty` flag is needed for this to run successfully (TODO: figure out / document why)
```

Comment:
I was curious about this and not sure if you've seen this but this made sense to me: https://stackoverflow.com/a/54793708

Reply:
Here's [a comment from a Heroku developer](https://git.io/JUBhj) explaining why this flag is necessary.